test/experimental/test_vectors.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+import torch
+
+from test.common.assets import get_asset_path
+from test.common.torchtext_test_case import TorchtextTestCase
+from torchtext.experimental.vectors import (
+    Vectors,
+    vectors_from_csv_file
+)
+
+
+class TestVectors(TorchtextTestCase):
```

Comment:
Should we test the jitability here? A JIT-able vector object should convert a token into vector on jit mode.

Reply:
Added a new test for this